ServerCore/Pages/Teams/AllTeams.cshtml

```diff
@@ -37,6 +44,20 @@ Registered Teams: <b>@Model.Teams.Count/@Model.Event.MaxNumberOfTeams</b>, Regis
                 <td>
                     @Html.DisplayFor(modelItem => @Model.PlayerCountByTeamID[item.ID])
                 </td>
+                @if (Model.EventRole == ModelBases.EventRole.admin ||
+                     Model.EventRole == ModelBases.EventRole.author)
+                {
+                    if (item.IsDisqualified)
+                    {
+                        <td>
```

Comment:
Seems like you could put the entire if statement inside the <td> so that it's obvious that this section is filling out the single cell?